Document how to turn on the extension feature (#55)
st0012/ruby-lsp-rspec: README.md

end
```

> Make sure the relevant features are enabled in `rubyLsp.enabledFeatures` in your VSCode settings, such as `codeLens`. See https://shopify.github.io/ruby-lsp for the full list of configuration options.

Reviewer: ```suggestion
> [!IMPORTANT] 
> Make sure the relevant features are [enabled](https://github.com/Shopify/ruby-lsp/tree/main/vscode#enable-or-disable-features) under your VSCode's `rubyLsp.enabledFeatures` setting, such as `codeLens`.
```

Author: Thanks! Committed
